appserver/monitoring-console/core/src/main/java/fish/payara/monitoring/store/InMemoryAlarmService.java

```diff
@@ -123,12 +126,12 @@ void changedConfig(boolean enabled) {
         if (!enabled) {
             checker.stop();
         } else {
-            checker.start(executor, 2, SECONDS, this::checkWatches);
+            checker.start(executor, 1, SECONDS, this::checkWatches);
```

Comment:
NB. Initially I thought evaluating alerts every two seconds is good enough. While this is true in general this does allow to miss value spikes that should cause an alert in case of most basic condition that only looks at the latest value. So it was changed to 1 second to make sure each value is considered.